bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -78,14 +79,22 @@
             description = "If true, the last modified value will be included in the sitemap.")
     private static final String PROP_INCLUDE_LAST_MODIFIED = "include.lastmod";
 
-    @Property(label = "Change Frequency Properties",
+    @Property(label = "Change Frequency Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the change frequency value.")
     private static final String PROP_CHANGE_FREQUENCY_PROPERTIES = "changefreq.properties";
 
-    @Property(label = "Priority Properties",
+    @Property(label = "Priority Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the priority value.")
     private static final String PROP_PRIORITY_PROPERTIES = "priority.properties";
 
+    @Property(label = "DAM Folder Property",
+            description = "The JCR property name which will contain DAM folders to include in the sitemap.")
+    private static final String PROP_DAM_ASSETS_PROPERTY = "damassets.property";
```

Comment:
Just so I'm clear, the way this works is that on the root page (or whatever page is being requested for the site map), this property get set to point to one or more asset folders. Makes sense, although I think it would be better to make this an array of property names, similar to the priority and change frequency ones.